### ISSUE TITLE
Group Field is Blank | 

### ISSUE THREAD

**Issue by iamrobert** [FLEXIcontent/flexicontent-cck#393]
I am trying to make a field group - however it doesn't display with FC 3.0.9-rc7   Release date: 2015-11-30.

Its blank:
![download](https://cloud.githubusercontent.com/assets/8617673/11505111/b576cb28-9883-11e5-9536-ee694ed402ef.png)

I tried enabling my FAQ version which worked in past versions - but it failed - and created this error:
Uncaught ReferenceError: addField63 is not definedonclick @ index.php?option=com_flexicontent&controller=items&task=items.add&typeid=10&252dcc7f04268e0695269ad…:2089

In the fields - they seem set up:
![download 1](https://cloud.githubusercontent.com/assets/8617673/11505281/9911101e-9884-11e5-8090-43dd883067d7.png)

and the code is blank:

```
<div style="" class="fcfield_row1 container_fcfield container_fcfield_id_84 container_fcfield_name_product_details" id="container_fcfield_84">

</div>

**Comment by ggppdk** [comment]
What is the field type of the fields inside each group
e.g. FAQ group has 2 text fields ?

**Comment by iamrobert** [comment]
Yes - 1. textfield + 1. textarea.

The other fieldgroup just textfield - no validation or anything

![fields kuei yih rubber latex administration](https://cloud.githubusercontent.com/assets/8617673/11507067/3e2407f2-988d-11e5-91fb-fc32166a6a90.png)

**Comment by iamrobert** [comment]
If I remove the field from the group it displays normally.

**Comment by ggppdk** [comment]
Ok the new code will remove groups that are ALL-empty
and also will not insert empty values into the DB

but it is supposed to allow: 
- at least 1 empty value in item form
but there seems to be an issue in the new logic, 
- either in the fieldgroup or inside the fields

i can replicate this, i am fixing it

**Comment by ggppdk** [comment]
indeed, the (at-least 1) empty group of value is not accounted properly, please test this fix:

- i say test because i get no labels, but your image shows the label, so i am not sure if there is 1 or 2 bugs in the value counting of fieldgroup

plugins/flexicontent_fields/fieldgroup/fieldgroup.php
replace:
```
if ($do_compact) $max_count -= count($grp_isempty);
```

replace:
```
if ($do_compact) $max_count -= (count($grp_isempty) - $start_at);
```

**Comment by iamrobert** [comment]
I just installed the most recent version and the problem has disappeared. Awesome! Thanks!

**Comment by ggppdk** [comment]
Great, thanks for testing fix,
Fixed with: d4467bb
